pkg/api/pod/util.go

```diff
@@ -733,3 +739,14 @@ func multiplePodIPsInUse(podStatus *api.PodStatus) bool {
 	}
 	return false
 }
+
+// hostnameFQDNInUse returns true if any pod's spec defines hostnameFQDN field.
+func hostnameFQDNInUse(podSpec *api.PodSpec) bool {
+	if podSpec == nil {
+		return false
+	}
+	if podSpec.HostnameFQDN {
```

Comment:
You can simplify these lines:

return podSpec.HostnameFQDN

Reply:
right, thanks